packages/api/src/apps/api/app.ts

```diff
@@ -72,9 +73,19 @@ async function run(env: ProcessEnv) {
     lastBlock: 0,
     lastBlockUpdate: 0,
     registeredEmps: new Set<string>(),
+    registeredLsps: new Set<string>(),
     collateralAddresses: new Set<string>(),
     syntheticAddresses: new Set<string>(),
+    // lsp related props. could be its own state object
+    longAddresses: new Set<string>(),
+    shortAddresses: new Set<string>(),
+    multicall: new Multicall(env.multicallAddress, provider),
+    lsps: {
+      active: lsps.JsMap("Active LSP"),
+      expired: lsps.JsMap("Expired LSP"),
+    },
```

Comment:
this is the global state, a bunch of new tables and data structures related to lsps are created. the new services require this as a dependency